spark-bigquery-connector-common/src/test/java/com/google/cloud/spark/bigquery/SparkBigQueryConfigTest.java

```diff
@@ -190,6 +190,29 @@ public void testConfigFromOptions() {
     assertThat(config.getBigQueryJobLabels()).containsEntry("foo", "bar");
   }
 
+  @Test
```

Comment:
Can you add another test to make sure an exception is thrown if a negative value is passed? Optionally, you can also add a test to make sure that a positive value passes